packages/lesswrong/components/tagging/SingleLineTagUpdates.tsx

```diff
@@ -161,7 +162,7 @@ const SingleLineTagUpdates = ({tag, revisionIds, commentCount, commentIds, users
           <span>History</span>
         </Link>}
       
-      {revisionIds.length>0 && <Link to={`revisions/tag/${tag.slug}`} className={classes.subheading}>
+      {revisionIds.length>0 && <Link to={`revisions/${tagUrlBaseSetting.get()}/${tag.slug}`} className={classes.subheading}>
```

Comment:
This route actually does hardcode `/revisions/tag/:slug`, rather than using any settings (even for the EA forum).